artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2790,6 +2790,14 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="auto-create-queues-durable" type="xsd:boolean" default="true" maxOccurs="1" minOccurs="0">
```

Comment:
please make this an enum, currently this is just true or false, and overrides the durablitly no matter what in code. (default behaviour should be existing behaviour)

Reply:
Please, would you be so patient and explain me, how to do this? I have implemented enum (in latest commit), but I don't think so it is what you intended.

Reply:
This is so that any override can be switched to OFF.